c7n/filters/health.py

```diff
@@ -65,23 +70,25 @@ def process(self, resources, event=None):
         return [resource_map[resource_id] for resource_id in found]
 
     def get_filter_parameters(self):
+        phd_svc_name_map = {
+            'app-elb': 'ELASTICLOADBALANCING',
+            'ebs': 'EBS',
+            'efs': 'ELASTICFILESYSTEM',
+            'elb': 'ELASTICLOADBALANCING',
+            'emr': 'ELASTICMAPREDUCE'
+        }
         m = self.manager
-        if m.data['resource'] == 'ebs':
-            service = 'EBS'
-        else:
-            service = m.get_model().service.upper()
+        service = phd_svc_name_map.get(m.data['resource'], m.get_model().service.upper())
         f = {'services': [service],
-             'regions': [self.manager.config.region],
+             'regions': [self.manager.config.region, 'global'],
```

Comment:
does 'global' region apply to individual resources, afaics all the regions this is supporting are regional?